### PR TITLE
[4.0] Fixed the style of nested lists and minor errors

### PR DESCRIPTION
**Pull request by vamera**
## Description

This PR is mainly focused on fixing the style for the nested list (sublists) such as the spacing and type.

In addition, I have included a fix for:
- the style of directives `versionadded`, `versionchanged`, `deprecated` and `seealso` within admonitions;
- the ordered list in `/docker/wazuh-container.html`;
- the hierarchy of the nested lists and the indentation of the `Note` in `/deploying-with-ansible/guide/install-ansible.html`;
- the directive `image` has been replaced with the directive `thumbnail` in `/user-manual/kibana-app/reference/custom-dashboard.html` for consistency of style and behavior;
- the lists in `/development/wazuh-cluster.html#integrity-thread`, which Sphinx was compiling as definition lists;
- the indentation and numbering in `/user-manual/kibana-app/reference/configure-indices.html`;
- the `Note` under step 5 in `/user-manual/kibana-app/reference/configure-indices.html`;
- the indentation of the lists in `/learning-wazuh/survive-flood.html`;
- the indentation and numbering in `/learning-wazuh/audit-commands.html`;
- the indentation of the lists in `/learning-wazuh/suricata.html` (and removed the empty code-block);
- the indentation of the list in `/docker/container-usage.html`;
- the indentation of the lists in `/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.html`;
- the indentation of the sublists in `/upgrade-guide/upgrading-elastic-stack/elastic_server_rolling_upgrade.html` and `/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.html`;
- the nested list in `/learning-wazuh/suricata.html#see-suricata-nids-events-in-kibana`, which Sphinx was compiling as definition list;

Related issue: https://github.com/wazuh/wazuh-website/issues/1385

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/deploying-with-ansible/guide/install-ansible.rst [hunk 1]
```diff
@@ -36,15 +36,15 @@ Installation on CentOS/RHEL/Fedora
 
 Install using yum from `EPEL <http://fedoraproject.org/wiki/EPEL>`_. Only CentOS/RedHat version 6 or 7, and Fedora distributions, are currently supported. Follow the next steps:
 
-1 - Install EPEL repository:
+1. Install EPEL repository:
```
Comment (davidfmiranda) [inline source/deploying-with-ansible/guide/install-ansible.rst:39]: Please use `#.` for enumerations

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @vamera, I've reviewed your PR and LGTM, nevertheless, I've suggested you changing the enumerations numbers by `#.`

Regards,

David

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Review by saramoran — APPROVED** [review]
LGTM

**Pull request merged**